nomenclature/definitions/variable/transport_types.yaml

```
@@ -0,0 +1,63 @@
+# This list of fuels will replace any occurrence of `<Transport>`
+# in a variable-definition yaml file
+
+<Transport>:
+  Maritime:
+     description: Martime transportation technologies
```

Typo (also in subsequent definitions)
```suggestion
     description: Maritime transportation technologies
```